docs/upgrading.md

```diff
@@ -0,0 +1,66 @@
+# Upgrading Contour to 0.15
+
+## Changes in 0.15
+
+### Breaking change
+The major breaking change in Contour 0.15 is that `contour serve` (the mode that actually serves the control plane for Envoy) now requires that either TLS certificates be available, or you supply the `--insecure` parameter.
+
+**Without one of these options, Contour will not start.**
+
+### Other changes
+
+With the increased security available with supplying TLS certificates for securing gRPC, we are recommending that users migrate to separate Contour and Envoy pods, deployed using a Deployment and Daemonset respectively. This will soon enable greater functionality for status updates on Contour-managed objects, and should help with upgrade issues eventually.
```

Comment:
Delete the editorial. This should be the how of how to go from 0.14 to 0.15, not the why.